### DISCUSSION TITLE
HLSL 2021 support?

### DISCUSSION THREAD

**Discussion by WangHoi** [DiligentGraphics/DiligentEngine#303] [General]
Will Diligent support HLSL 2021 in near future?
I'm really looking forward to use the operator overloading features.
It can reduce lots of boiler code, when implement dual-complex number related algorithms.

It seems that DXC support it, but the HLSL2GLSL translator does not support HLSL 2021.

**Comment by TheMostDiligent** [comment]
You should already be able to use HLSL 2021 with DXC compiler with D3D12 and Vulkan backends.
OpenGL is deprecated and nobody should use it. HLSL2GLSL translator will not support HLSL 2021.
